.jenkins/pytorch/test.sh

```diff
@@ -19,6 +19,11 @@ if [[ "$BUILD_ENVIRONMENT" == *-slow-* ]]; then
   export PYTORCH_TEST_SKIP_FAST=1
 fi
 
+if [[ "$BUILD_ENVIRONMENT" == *old-gradcheck* ]]; then
+  export PYTORCH_TEST_WITH_SLOW_GRADCHECK=ON
+  export PYTORCH_TEST_WITH_SLOW=1
```

Comment:
Not sure we want to run the slow tests here? In particular, running the fast + slow test will most likely result in a timeout because it will take too long.